sentry-android-core/src/main/java/io/sentry/android/core/internal/gestures/SentryGestureListener.java

```diff
@@ -163,9 +169,13 @@ private void addBreadcrumb(
       className = target.getClass().getSimpleName();
     }
 
+    final Map<String, Object> hintMap = new HashMap<>();
+    hintMap.put("MotionEvent", motionEvent);
```

Comment:
can we also put the `target` to the hintMap? Might be useful to retrieve some View's attributes